src/assets/json/prescriber-form.json

```diff
@@ -141,12 +141,13 @@
         "placeholder": "Enter your office contact phone #",
         "type": "text",
         "subType": "phone",
+        "class": "office-contact-phone-position",
         "validators": { "min": 14, "max": 14 }
       },
       {
         "name": "officeContactExt",
```

Comment:
remove this. why do u need this. is this still being used for creating form and then to set value?